src/Components/Search/SearchBar.tsx

```diff
@@ -252,9 +263,6 @@ export class SearchBar extends Component<Props, State> {
     if (!focused) {
       return null
     }
-    if (xs && !query) {
```

Comment:
Should this have been removed? I seem to remember us being confused that this was in the diff.

Reply:
No! Thank you for catching that. I'm not sure why this keeps happening, but I'll remove it.